### PR TITLE
Feat/enable tron staking via stakekit backport

### PR DESCRIPTION
**Pull request by hedi-edelbloute**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** tron staking discover button
  - ...

### 📝 Description

Backport of [https://github.com/LedgerHQ/ledger-live/pull/6283](https://github.com/LedgerHQ/ledger-live/pull/6283) from stakekit

### ❓ Context

- **JIRA or GitHub link**: [https://ledgerhq.atlassian.net/browse/LIVE-11441](https://ledgerhq.atlassian.net/browse/LIVE-11441)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by mcayuelas-ledger — DISMISSED** [review]

File: apps/ledger-live-desktop/src/renderer/families/tron/Votes/index.tsx [hunk 1]
```diff
@@ -113,7 +169,14 @@ const Delegation = ({ account }: { account: TronAccount }) => {
               ) : null
             }
           >
-            <Button disabled={true} color="palette.primary.main" small>
+            <Button
+              onClick={
+                hasRewards && canClaimRewards ? () => claimOnClick(tronStakingProvider) : noop
```
[the remark this answers is not in the document]
Comment (hedi-edelbloute) [reply apps/ledger-live-desktop/src/renderer/families/tron/Votes/index.tsx:174]: @Philippoes WDYT ?

File: .changeset/soft-snakes-prove.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
```
[the remark this answers is not in the document]
Comment (hedi-edelbloute) [reply .changeset/soft-snakes-prove.md:2]: @Philippoes

File: apps/ledger-live-desktop/src/renderer/families/tron/Votes/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Philippoes) [reply apps/ledger-live-desktop/src/renderer/families/tron/Votes/index.tsx:174]: Great catch. Will submit a PR to this branch with the fix.

**Review by sprohaszka-ledger — APPROVED** [review]

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**